Clean Swift/Scene.xctemplate/UITableViewController/___FILEBASENAME___Interactor.swift

```diff
@@ -14,15 +14,15 @@ protocol ___FILEBASENAMEASIDENTIFIER___InteractorOutput {
 
 class ___FILEBASENAMEASIDENTIFIER___Interactor: ___FILEBASENAMEASIDENTIFIER___ViewControllerOutput {
     var output: ___FILEBASENAMEASIDENTIFIER___InteractorOutput!
-    var worker: ___FILEBASENAMEASIDENTIFIER___Worker!
+    var store: ___FILEBASENAMEASIDENTIFIER___WorkerProtocol!
 
     // MARK: - Business logic
 
     func doSomething(request: ___FILEBASENAMEASIDENTIFIER___Request) {
         // NOTE: Create some Worker to do the work
 
-        worker = ___FILEBASENAMEASIDENTIFIER___Worker()
-        worker.doSomeWork()
+//        store = ___FILEBASENAMEASIDENTIFIER___Worker()
```

Comment:
I've removed the comments, but can't say I'm happy with the following call on `store`. On the other hand removing it will leave the method kind of empty and useless. Any suggestions?